app/config/production.json

```diff
@@ -0,0 +1,4 @@
+{
+  "OPENSHIFT_APP_NAMESPACE": "ff61fb-prod",
+  "SITEMINDER_LOGOUT_URL": "https://logontest7.gov.bc.ca/clp-cgi/logoff.cgi"
```

Comment:
Should there be a different logout url for prod?

Reply:
Good point. Is it https://logon.gov.bc.ca/clp-cgi/logoff.cgi ?